mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/route/DocumentRouteDescriptor.java

```diff
@@ -21,6 +21,10 @@ private DocumentRouteDescriptor(String routingKey) {
 		this.routingKey = routingKey;
 	}
 
+	public String routingKey() {
+		return routingKey;
+	}
+
```

Comment:
I don't know why you moved this up here, but for the sake of understanding each other I'll clarify why I put it down there in the first place.

I generally try to order methods this way: start with implemented and overridden methods, from the highest classes/interfaces in the hierarchy, then new methods declared by this class.

That's not a rule, but, well, next time you see this and feel the urge to change it, you'll know it wasn't random ;)

Reply:
I used to put hashCode, equals and toString after other methods... sorry if I changed it :P 
Going to revert it...